edn/src/edn.rustpeg

```diff
@@ -12,9 +12,11 @@
 
 use std::collections::{BTreeSet, BTreeMap, LinkedList};
 use std::iter::FromIterator;
+
```

Comment:
Yeah, I decided to do much what we've been doing in Swift and Java: split std out, sort all alphabetically within that division.